wg-advisors/README.md

```diff
@@ -102,4 +102,37 @@ the ASF. Trust is transitive - that is, you may not know all members
 personally, but each member was nominated and elected by people that
 you, in turn, nominated and elected.
 
+### How to initiate PMC <-> Advisor relationship?
 
+There is no formal process for this. As an advisor, you may simply
+start participating in the project's mailing lists (dev and private)
+after introducing yourself and asking if this is fine for you to become
+an Advisor. The PMC must be allowed to say "no thank you" without providing any
+reason, and you must respect that decision, and not offer again unless
+invited.
+
+You can also be invited by the PMC to become an Advisor. When you become
+an Advisor, you should sign yourself up to the list of advisors in this
+repository (TODO: how) and describe your experience, skills and what
+you can bring as an Advisor (TODO: how).
+
+The ASF Board can also - during regular Board project review process - to
+identify a project that could benefit from an Advisor, and ask the PMC
+to consider inviting an Advisor, and cc: advisors mailing list (TODO: add the list)
```

Comment:
I am not persuaded that we need a new mailing list for this. I see this WG as one of the core functionalities of the ComDev PMC, and moving that discussion off to another list seems a way to remove it from view, with no particular benefit. I'd ask that we keep it in the main list until such time that people actually complain that there's too much noise, since I think we can all learn from these discussions.

Reply:
Fine with that as well. I think having it all on comdev list is great.

Reply:
I think I addressed it